src/squid.md

```diff
@@ -0,0 +1,247 @@
+---
+title: squid
+section: 8
+---
+# NAME
+
+**squid** - HTTP web proxy caching server
+
+# SYNOPSIS
+
+**squid** [-dsCFNRSXY] [-\-foreground] [-l facility] [-f file] [-[au] port] [-k signal] [-n service]
+
+**squid** -[hv]
+
+**squid** -n service -i [-dsX] [-\-foreground] [-l facility]
+
+**squid** -n service -r [-dsX] [-\-foreground] [-l facility]
+
+**squid** -n service -O command [-ds] [-\-foreground] [-l facility]
+
+**squid** -z [-NS] [-dsX] [-\-foreground] [-l facility] [-f file] [-n service]
+
```

Comment:
It looks like this PR is trying to enumerate some or possibly all mutually exclusive ways to run "squid" in synopsis. IMO, there are too many of them and/or the enumeration principles are too complex -- the end result does not work well as a quick summary of how to start Squid. It would also be rather difficult to maintain as developers will forget to update some of these lines when adding options (among other potential problems).

I suggest simplifying the whole thing to just say something like:

```
# SYNOPSIS

**squid** [option]...
```

or

```
# SYNOPSIS

**squid** [options]
```

... and let individual option documentation to detail option compatibility, requirements, and other important caveats.